PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3282,8 +3282,6 @@ void Notepad_plus::dropFiles(HDROP hdrop)
 
 		if (isOldMode || folderPaths.size() == 0) // old mode or new mode + only files
 		{
-
-			BufferID lastOpened = BUFFER_INVALID;

```

Comment:
Instead of scope change, I'd remove Line 3256 & Line 3316-3319 due to being unused and redundant.